plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -840,8 +841,12 @@ public boolean isTableCurrent(ConnectorSession session, ConnectorTableHandle tab
         return tableToken.get().getSnapshotId() == currentToken.get().getSnapshotId();
     }
 
+    /**
+     * @throws MaterializedViewNotFoundException when materialized view is not found
```

Comment:
Remove the javadoc here, from the implementation, add this to interface declaration instead